lib/floe.rb

```diff
@@ -45,7 +45,27 @@ def self.logger
     @logger ||= NullLogger.new
   end
 
+  # Set the logger to use
+  #
+  # @example
+  #   require "logger"
+  #   Floe.logger = Logger.new($stdout)
+  #
+  # @param logger [Logger] logger to use for logging actions
   def self.logger=(logger)
     @logger = logger
   end
+
+  # Set the runner to use
+  #
+  # @example
+  #   Floe.set_runner "docker", kubernetes", {}
```

Comment:
super tiny typo here

```suggestion
  #   Floe.set_runner "docker", "kubernetes", {}
```